### PR TITLE
docs: Updated Quotes/Rates #1977

### PR DESCRIPTION
**Pull request by Cioraz**
## Changes proposed in this pull request

Updated and fixed issue #1977 

## Context

Edited Docs and fixed minor changes.
More edits on the content for the docs are welcome.

## Checklist

- [ ] Related issues linked using `fixes #number`
- [ ] Tests added/updated
- [x] Documentation added
- [ ] Make sure that all checks pass
- [ ] Postman collection updated

### CODE REVIEW

File: packages/documentation/src/content/docs/integration/getting-started.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ Account Servicing Entities provide and maintain payment accounts. In order to ma
 
 Furthermore, each payment account managed by the Account Servicing Entity needs to be issued at least one [payment pointer](#issuing-payment-pointers) in order to be serviced by Rafiki and send or receive Interledger payments.
 
-## Quotes / Rates and Fees
+## Quotes and Fees
 
-Every Interledger payment is preceded with a quote that estimates the costs for transfering value from A to B. The Account Servicing Entity may set fees on top of that for facilitating that transfer. How they structure those fees is completely up to the Account Servicing Entity.
+Every Interledger payment is preceded with a quote that estimates the costs for transfering value from A to B. The Account Servicing Entity may set fees on top of that for facilitating that transfer. How they structure those fees is completely up to the Account Servicing Entity. The fees are entirely set by the asset
```
Comment (sabineschaller) [inline packages/documentation/src/content/docs/integration/getting-started.md:20]: > The fees are entirely set by the asset

Unfortunately, this is not correct. 

```suggestion
Every Interledger payment is preceded with a quote that estimates the costs for transferring value from A to B. The Account Servicing Entity may charge fees on top of that for facilitating that transfer. They can specify fixed and variable fees per asset using the Admin API. How they structure those fees is completely up to the Account Servicing Entity.
```

**Review by sabineschaller — COMMENTED** [review]
Hello and thank you for the contribution!

Our CI pipeline requires the title of a PR to be [conventional](https://www.conventionalcommits.org/en/v1.0.0/). Please add a "docs:" in the beginning.

**Review by sabineschaller — APPROVED** [review]
Thank you for contributing!

**Comment by devcer** [comment]
Thank you for Contributing to Interledger during Hacktoberfest. If you have any accepted PRs, You can claim your Swag [here](https://community.interledger.org/devcer/hacktoberfest-swag-redemption-claim-your-cool-gear-2152). Last date to fill up the form is December 10th, 2023. @Cioraz

**Pull request merged**